src/services/firebase.js

```diff
@@ -0,0 +1,15 @@
+import firebase from 'firebase'
+require('firebase/auth')
+
+/**
+ * Initialize Firebase application with parameters
+ */
+const firebaseConnection = firebase.initializeApp({
+    apiKey: "AIzaSyBziUug99HMl-cwPOzWnwHnzJxH_BmG-N8",
```

Comment:
Uhhh should api keys really be uploaded to github and not passed around by other, more private means?

Reply:
we could store in an .env file but then people can just open the .env file... is there any solution you know that is good for this?

Reply:
also, given this is a private repo, I don't see it as too much of an issue?

Reply:
Oh, it's been set up as a public repo...

Reply:
If Firebase isn't complaining and revoking the access rights to the keys, then I would say leave it for now :P. I have had a few experiences with Google Revoking my API keys due to accidental pushes onto Github.

However, if we want to address this, then git-secret is a solution I think.